src/js/util/param_spec.js

```diff
@@ -0,0 +1,40 @@
+function ParamSpec(sourceLow, sourceHigh, distLow, distHigh, clamp = true) {
+  const domain = sourceHigh - sourceLow
+  const range = distHigh - distLow
+  let map
+  let unmap
+
+  if (clamp) {
```

Comment:
> Do we use clamp? If no, why add it?

yeah, it clamps by default which is how it's used [here](https://github.com/morons-llc/synth-frontend/pull/4/files#diff-5c0e44c2d96f70060565edab9aed2d62R6)